src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/furtherevidence/reissuefurtherevidence/ReissueFurtherEvidenceAboutToSubmitHandlerTest.java

```diff
@@ -181,7 +188,7 @@ public void returnsAnErrorIfItCouldNotFindTheSelectedDocumentToReIssueFurtherEvi
 
     @Test
     public void returnsAnErrorIfReIssuedToRepresentativeWhenThereIsNoRepOnTheAppealToReIssueFurtherEvidence() {
-        sscsCaseData = sscsCaseData.toBuilder().resendToRepresentative("YES").build();
+        sscsCaseData.getReissueArtifactUi().setResendToRepresentative(YesNo.YES);
```

Comment:
There are no tests for other parties. Add one in. Also perhaps for ReissueUtils.validateSelectedPartyOptions

Reply:
There is, see givenCaseWithMultipleOtherParties_thenBuildTheOtherPartyOptionsSection